scripts/dev.ts

```diff
@@ -303,7 +303,7 @@ async function main() {
     },
     {
       type: 'transfer',
-      timestamp: moment().subtract(26, 'minutes').unix(),
+      timestamp: moment().subtract(46, 'minutes').unix(),

```

Comment:
Lets add a comment here stating what we found, something like:
"We use this to reset to local time, in future we should ensure the automatic process works"